src/Login/Logout.js

```diff
@@ -6,32 +6,7 @@ import Cookies from 'universal-cookie';
 const cookies = new Cookies();
 
 export default function Logout(){
-    useEffect(() => {
-        const data= {
-            "email": cookies.get(),
-        }
-        
-        call(`/clients/logout}`,"POST",data)
-            .then(async response  =>  {
-
-            if(response.ok && await response.json()==="SUCCESS"){
-                cookies.remove('email', {path: "/"});
-                window.location.href='./login';
-            }else{
-                alert("no se ha podido deslogar")
-            }
-            return response.data;
-        })
-        .catch(error=>{
-            console.log(error);
-        })
-
- 
-    }, []);
-
-    return (
-        <div className='App'>
-        </div>
-    )
-    
+    cookies.remove('AuthToken'); // determine if authorized, from context or however you're doing it
+    localStorage['AuthToken'] = undefined
+    window.location.href='./login';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9156896)
```suggestion
    window.location.href="./login";
```